server/views/partials/additionalConditionsInputs/placeOfResidence.njk

```diff
@@ -0,0 +1,12 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{{ govukInput({
+    id: 'regionOfResidence',
+    name: params.id + '[regionOfResidence]',
+    classes: "govuk-!-width-one-third",
+    label: {
+        text: "Enter probation region"
+    },
+    errorMessage: params.errorMessages | findError('regionOfResidence'),
+    value: params.formResponses[params.id].regionOfResidence
+}) }}
```

Comment:
Its not that obvious what is contained within `params` here? I wonder if to keep things clear we could add a comment at the top of each additional condition template to describe what is expected to be provided to it? Might make this easier to maintain/ change in future?
e.g.
`{%
   This template expects a params object containing:
      id - the condition identifier
      y - xxxx
%}`